### PR TITLE
Home

### PR DESCRIPTION
**Pull request by poojavishnoi**
Made home page and made a structure for the project.
Browse this preview: (https://deploy-preview-1--neodevstore.netlify.app/)

### CODE REVIEW

**Review by sonishreyas — APPROVED** [review]
The code looks clean and easy to understand.

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,20 @@
 import "./App.css";
-import logo from "./logo.png";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Header from "./components/Header";
+import Home from "./pages/Home";
+import Footer from "./components/Footer";
 
 function App() {
   return (
+    <Router>
```
Comment (ravikumar0403) [inline src/App.js]: If you use `Router` in App.js then you cannot use any router hooks in App.js which might be required later, So you can use `Router` in index.js

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  return (
+    <div className="navigation_main_container">
+      <div className="nav_section">
+        <div className="burger">
+          <i className="fas fa-bars"></i>
+        </div>
+        <div className="brand_logo">
+          <Link to="/">
+            <img src={"https://cdn.pixabay.com/photo/2016/12/07/15/15/lotus-with-hands-1889661_960_720.png"} alt="" />
```
Comment (ravikumar0403) [inline src/components/Header.js:14]: `{ }` are not required if it's just a string

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+
+import { products } from "../backend/db/products";
```
Comment (ravikumar0403) [inline src/pages/Home.js]: You should make an api call to get the products from backend instead of importing it this way

File: src/pages/Home.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        </div>
+      </div>
+      <div className="heading">
+        <h2>CATEGORY</h2>
+      </div>
+
+      <div className="category_grid">
+        <div className="category">
+          <Link to="/productlist/dress">
+            <img className="img_responsive" src={"https://images.pexels.com/photos/7506739/pexels-photo-7506739.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260"} alt=""></img>
```
Comment (ravikumar0403) [inline src/pages/Home.js]: here as well `{ }`  are not required

File: src/style/header.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
+.navigation_main_container {
+  border: none !important;
```
Comment (ravikumar0403) [inline src/style/header.css]: Can this be done without `!important`?

File: src/style/header.css [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+.brand_title {
+  display: block !important;
```
Comment (ravikumar0403) [inline src/style/header.css]: same here, try to avoid `!important`

**Review by ravikumar0403 — COMMENTED** [review]
Code is clean and readable, I've left a few comments which you can implement

File: src/App.js (hunk 1 above)
Comment (poojavishnoi) [reply src/App.js]: I didn't knew about this.Thank you

File: src/components/Header.js (hunk 2 above)
Comment (poojavishnoi) [reply src/components/Header.js:14]: ok

File: src/style/header.css (hunk 5 above)
Comment (poojavishnoi) [reply src/style/header.css]: done

File: src/style/header.css (hunk 6 above)
Comment (poojavishnoi) [reply src/style/header.css]: done

Comment (poojavishnoi) [reply src/style/header.css]: Thank you for all the suggestion, made the required changes

**Pull request merged**